### ISSUE TITLE
[1.16.5] Diesel Generators running unnecessarily

### ISSUE THREAD

**Issue by Kaligraphy22** [BluSunrize/ImmersiveEngineering#4820]
### Description of the issue:
I posted about this issue in 1.15.2, I think. The generators are running even when the capacitor it's connected to is full.

### Crashlog:
N/A

### Versions & Modlist
forge-1.16.5-36.1.13-installer
Mod list attached.
[Mod List.txt](https://github.com/BluSunrize/ImmersiveEngineering/files/6488392/Mod.List.txt)

**Comment by BluSunrize** [comment]
No, the generators run if there is even the tiniest amount of space in the target capacitor.
If you want to prevent that, build yourself some redstone automation using a comparator.

**Comment by Kaligraphy22** [comment]
Ok so I unhooked everything from the capacitor, turned off (redstone signal) all the machines and tested all my machines one after another. I did this cause something was indeed constantly draining power from the HV capacitor which is why the diesel generator continued to run.

The thing is, after I hooked everything back up one by one (and tested whether it was draining power when it shouldn't), there was no random power drain. I suspect there one of my garden cloches was running but I don't know for sure now. So yes, you were 100% correct and I apologize for not fully testing stuff before I opened an issue. I thought I was sure on this but obviously I wasn't.
